quantum/quantum.c

```diff
@@ -49,6 +49,10 @@ extern backlight_config_t backlight_config;
 #    include "encoder.h"
 #endif
 
+#ifdef WPM_ENABLE
+#    include "wpm.h"
+#endif
```

Comment:
This block should be moved to `quantum.h` instead.

Reply:
The header is currently included in `quantum.h` as well.  I thought it was a little strange to see many header includes for features duplicated in `quantum.c` (velocikeys, haptic, backlight, fauxclicky, etc), but I sort of followed along thinking that someone smarter than me knew something I did not :-)

I'll remove it from `quantum.c` and re push to my branch.

Reply:
A lot of those are legacy, I think. Or ... other people following existing code, too.  😆